File: sentimental-mario-less/mario.py
from cs50 import get_int


while(True):
    tamanho = get_int("Height: ")
    if(tamanho >= 9 or tamanho < 1 or tamanho == str or tamanho == []):
        continue
    else:
        break


espaco = tamanho - 1
for i in range(0, tamanho):
    print(((tamanho -1 -i) * ' '), end = '')
    print((i+1)*'#')

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, linha;
    do{
        height = get_int("Height: ");
    }
    while(height> 8 || height < 1);

    for(linha = 0; linha < height; linha++){
        for(int espaco = 0; espaco < height - linha - 1; espaco++){
            printf(" ");
        }
        for(int coluna = 0; coluna <= linha; coluna++){
            printf("#");
        }
        for(int separa = 0; separa <2;separa++){
            printf(" ");
        }
        for(int i = 0; i<=linha; i++){
            printf("#");
        }
        printf("\n");
    }

}